### PR TITLE
(on getgov-meoward) Ticket #1510: Fix table alignment

### LINKED ISSUE #1510: Fix alignment on dashboard tables
### Issue description

The domain and domain requests table columns don't align well. Add a 220 `width` to the first columns and bump the right padding to the first column only to `1.5 rem.` 

image
image

### Acceptance criteria

- [ ] columns start at the same position on the page and are aligned.

### Additional context

_No response_

### Links to other issues

🔄 to #1492

### PR DESCRIPTION
**Pull request by zandercymatics**
## Ticket

Resolves #1510

## Changes

- Sets table at a fixed width when the resolution is greater than 1040px

## Context for reviewers
This PR moves the alignment of the top and bottom tables on the main page such that they match when the resolution is greater than 1040px

## Setup
1. Login to the app, go to the home page
2. Note the alignment

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Comment by rachidatecs** [comment]
Have not tested yet but I assume you'll need overflow styles otherwise long words will break out of the table cells, @zandercymatics

File: src/registrar/assets/sass/_theme/_tables.scss [hunk 1]
```diff
@@ -108,12 +108,51 @@
...
+      width: 95px;
+    }
+
+    th:nth-of-type(5) {
+      width: 85px;
+    }
+  }
+}
+
+@media (min-width: 1040px){
```
Comment (rachidatecs) [inline src/registrar/assets/sass/_theme/_tables.scss:140]: No need to repeat the media query, it matches the one above

**Review by rachidatecs — COMMENTED** [review]
Will test next, it's on my list

**Review by therealslimhsiehdy — APPROVED** [review]
LGTM! TIL I learned about tables and nth type -- thanks for pointing these things out!

**Previously:**
Screenshot 2024-04-24 at 1 10 29 PM

**Now:**
Screenshot 2024-04-24 at 1 10 24 PM

**Comment by rachidatecs** [comment]
> > Have not tested yet but I assume you'll need overflow styles otherwise long words will break out of the table cells,
> 
> White space isn't set to nowrap so this is a non-issue. Our tables worked this way before with longer words so I think it should be OK unless design wants that

Don't merge yet, I'm not convinced. Did you test with a super long word?

**Comment by rachidatecs** [comment]
Just tested and your'e correct in your surmise, sir

**Review by rachidatecs — APPROVED** [review]

**Pull request merged**